tests/js/spec/views/organizationGroupDetails/__snapshots__/groupSimilar.spec.jsx.snap

```diff
@@ -317,8 +317,6 @@ exports[`Issues Similar View renders with mocked data 1`] = `
               </span>
               <QueryCount
                 count={4}
-                hideIfEmpty={true}
```

Comment:
This seems to be changing the behavior

Reply:
Is it possible, that it removes that prop because it would be redundant? Because if `hideIfEmpty` is not specified it will default (as specified in default args) to true anyway?

Reply:
Ah you're right, my bad!